pkg/apis/core/v1alpha1/constants/types_constants.go

```diff
@@ -97,12 +97,10 @@ const (
 	// StatefulSetNameAlertManager is a constant for the name of a Kubernetes stateful set object that contains
 	// the alertmanager pod.
 	StatefulSetNameAlertManager = "alertmanager"
-	// StatefulSetNameETCDMain is a constant for the name of a Kubernetes stateful set object that contains
```

Comment:
Can we keep those old constants and use it in other files where we still make assumptions that a `StatefulSet` is the result of this `Etcd` resource? E.g., https://github.com/gardener/gardener/pull/1869/files#diff-398162f3b552a38fdda6010a922a5998R497

Reply:
OK, just discussed with @georgekuruvillak, the health checks or any direct usage of the `StatefulSet` will be removed as part of #1762. Instead, the care controller will fetch the health status information from the `status` of the `Etcd` custom resource.
/lgtm